### PR TITLE
Add containing section on Modifying Behavior with Link Relationships

### PR DESCRIPTION
**Pull request by gkellogg**
…and add an Alternate Document Location section.

For #204.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/pull/211.html" title="Last updated on Aug 14, 2019, 4:33 PM UTC (dc5855e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-syntax/211/6dc0e7e...dc5855e.html" title="Last updated on Aug 14, 2019, 4:33 PM UTC (dc5855e)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10856,12 +10856,31 @@ <h3>Graph Containers</h3>
...
+
+  <p>When processing normal JSON documents, a link relation can be specified using
+    the <a>HTTP Link Header</a>
+    returned when fetching a remote document, as described in <a href="#interpreting-json-as-json-ld" class="sectionRef"></a>.</p>
+
+  <p class="changed">In other cases, a resource may be returned using a representation that cannot easily be interpreted
+    as JSON-LD. Normally, <a href="https://en.wikipedia.org/wiki/Content_negotiation">HTTP content negotiation</a>
+    would be used to allow a client to specify a preference for JSON-LD over another representation,
+    but in certain situations, it is not possible for a server to respond appropriately to such requests.
+    For the, an <a>HTTP Link Header</a> can be used to provide an alternate location for a document
```
Comment (rubensworks) [inline index.html]:
```suggestion
    For this, an <a>HTTP Link Header</a> can be used to provide an alternate location for a document
```

File: index.html [hunk 2]
```diff
@@ -10856,12 +10856,31 @@ <h3>Graph Containers</h3>
...
+    <a href="https://en.wikipedia.org/wiki/Link_relation" data-no-xref>Link Relation</a>.</p>
+
+  <p>When processing normal JSON documents, a link relation can be specified using
+    the <a>HTTP Link Header</a>
+    returned when fetching a remote document, as described in <a href="#interpreting-json-as-json-ld" class="sectionRef"></a>.</p>
+
+  <p class="changed">In other cases, a resource may be returned using a representation that cannot easily be interpreted
+    as JSON-LD. Normally, <a href="https://en.wikipedia.org/wiki/Content_negotiation">HTTP content negotiation</a>
+    would be used to allow a client to specify a preference for JSON-LD over another representation,
+    but in certain situations, it is not possible for a server to respond appropriately to such requests.
```
Comment (pchampin) [inline index.html]:
```suggestion
    but in certain situations, it is not possible or practical for a server to respond appropriately to such requests.
```

File: index.html [hunk 3]
```diff
@@ -10932,6 +10951,63 @@ <h3>Graph Containers</h3>
     ignored for such documents.</p>
 </section>
 
+<section class="normative changed"><h3>Alternate Document Location</h3>
+
+  <p>Documents which can't be directly interpreted as JSON-LD can provide an alternate location containing JSON-LD.
+    One way to provide this is by using referencing a JSON-LD document in an <a>HTTP Link Header</a>.
```
Comment (pchampin) [inline index.html]:
```suggestion
    One way to provide this is by referencing a JSON-LD document in an <a>HTTP Link Header</a>.
```

File: index.html (hunk 2 above)
Comment (gkellogg) [reply index.html]: Fixed in f1f4efe0c995b9ab901ed19cffdf7d286bd3961b.

File: index.html (hunk 3 above)
Comment (gkellogg) [reply index.html]: Fixed in f1f4efe0c995b9ab901ed19cffdf7d286bd3961b.

**Pull request merged**